apps/portal-api/src/server/mail-template/opencti_platform_unregistered.html

```diff
@@ -0,0 +1,330 @@
+<!doctype html>
+<html
+  lang="en"
+  xmlns:v="urn:schemas-microsoft-com:vml">
+  <head>
+    <meta charset="utf-8" />
+    <meta name="x-apple-disable-message-reformatting" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1" />
+    <meta
+      name="format-detection"
+      content="telephone=no, date=no, address=no, email=no, url=no" />
+    <meta
+      name="color-scheme"
+      content="light dark" />
+    <meta
+      name="supported-color-schemes"
+      content="light dark" />
+    <!--[if mso]>
+      <noscript>
+        <xml>
+          <o:OfficeDocumentSettings
+            xmlns:o="urn:schemas-microsoft-com:office:office">
+            <o:PixelsPerInch>96</o:PixelsPerInch>
+          </o:OfficeDocumentSettings>
+        </xml>
+      </noscript>
+      <style>
+        td,
+        th,
+        div,
+        p,
+        a,
+        h1,
+        h2,
+        h3,
+        h4,
+        h5,
+        h6 {
+          font-family: 'Segoe UI', sans-serif;
+          mso-line-height-rule: exactly;
+        }
+      </style>
+    <![endif]-->
+    <title>
+      OpenCTI Successfully Linked to XTM Hub – Integration Now Active
+    </title>
+    <style>
+      @media (max-width: 600px) {
+        .sm-my-8 {
+          margin-top: 32px !important;
+          margin-bottom: 32px !important;
+        }
+        .sm-px-4 {
+          padding-left: 16px !important;
+          padding-right: 16px !important;
+        }
+        .sm-px-6 {
+          padding-left: 24px !important;
+          padding-right: 24px !important;
+        }
+        .sm-leading-8 {
+          line-height: 32px !important;
+        }
+      }
+    </style>
+  </head>
+  <body
+    style="
+      margin: 0;
+      width: 100%;
+      background-color: #f1f5f9;
+      padding: 0;
+      -webkit-font-smoothing: antialiased;
+      word-break: break-word;
+    ">
+    <div
+      role="article"
+      aria-roledescription="email"
+      aria-label="OpenCTI Successfully Linked to XTM Hub – Integration Now Active"
```

Comment:
```suggestion
      aria-label="OpenCTI Successfully Linked to XTM Hub – Integration Now Active"
```

This needs to change too right ?